notebooks/geo/DEM_MapVisualization.ipynb

```diff
@@ -0,0 +1,215 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.        xml_urls.append([prod['properties']['ops:Label_File_Info.ops:file_ref'][0]])
</pre><p>It sounds like the order of the images differs depending on the request run. Maybe you should create a dictionnary with the lidvid as a key:</p><p> </p><p>xml_urls = {prod['properties']['lidvid'][0]:prod['properties']['ops:Label_File_Info.ops:file_ref'][0] for prod in resp_data}</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NASA-PDS/search-api-notebook/pull/28/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>